.gitignore

```diff
@@ -106,15 +106,30 @@ dmypy.json
 
 # End of https://www.gitignore.io/api/flask
 
+<<<<<<< HEAD
 ## Intellij ###
 .idea/
 
+=======
+>>>>>>> my commit
 ### macOS ###
 .DS_Store
 
 ### Virtual environment ###
 env/
+<<<<<<< HEAD
 .venv/
 
 ### Configuration files ###
 .env
+=======
+
+### Configuration files ###
+.env
+.idea/
+.vscode/
+.venv
+.vagrant
+pytestdebug.log
+geckodriver.log
```

Comment:
You should also gitignore geckodriver.exe:
 - it's a large file you're not responsible for
 - it needs to match your version of Firefox
 - You could have geckodriver on your PATH rather than in this folder